### DISCUSSION TITLE
Help needed to reverse effects of canbus hack

### DISCUSSION THREAD

**Discussion by ghost** [collin80/SavvyCAN#668] [Q&A]
Hi,

Some help is much needed and appreciated. I'm a 45 year old disabled man that lives in Faringdon, Oxfordshire. The disabilities are severe sensory PTSD, agoraphobia and psychogenic hyperventilation.

Since earlier this year the motability Golf has been hacked by people with anti-disability vendettas. They've accessed the canbus remotely so the suspension wallows, shudders and porpoises over undulations. Extensive research was conducted and I'm desperate for the hacking to stop. It's ruining what little quality of life there is.

Can Savvy seems to be able to access various canbus to alter settings? Please can you advise how it can be used to reverse the affects to the suspension? 

They've also altered another canbus so there's excessive noise and vibration through pedals and steering. Dealer resets haven't worked. All help is much appreciated. It would be much appreciated if we could arrange a video call so I can be shown how to address the issues.

Many thanks in advance,

Josh Ross

**Comment by simotronic** [comment]
I'm so sorry to hear about this kind of abuse which I have described as
digital gaslighting. Sadly the victims are often derided or rediculed when
they identify these situations.
The extreme connectedness of vehicles is also of concern given the amount
of tracking data that is available.
With respect to hacking the CANbus it has been demonstrated that CANbus
devices which don't have a secure key to encrypt write transactions and
some read transactions, are susceptible to hack attacks. Additionally
simply broadcasting electrical noise to interfere with sensors and the
system as a whole. Most of this type of hacking is downright dangerous
especially as some of the systems such as steering by wire can be
compromised.
Since the connected vehicles can be remotely accessed, a quick immediate
fix can be to disconnect the antenna (aerial, Josh). A smart phone being
substituted for navigational aid purposes.
I suspect military grade vehicles have hardened systems witch extensive
shielding of these subsystems. Of course auto manufacturers aren't likely
to want to add that expense... (Loss of profit). However, in EVs we have to
realize that the many HV harnesses are also susceptible to vermin attacks.
I will discuss this topic with my EV colleagues and my data communication
experts.

Thank you Collin for all the hard work you contribute to this important
part of transportation engineering.
Simon Gibson

**Reply by ghost** [reply]
Thank you for the reply Simon and for recognising gaslighting - it's prevalent and disturbing. The hack most definitely is happening and isn't imagined. 

The car is a 2021 diesel mk8 Golf Gtd. It's a motability car and vital for getting around and staying connected with family and friends. 

Severe sensory PTSD, agoraphobia and psychogenic hyperventilation means much of life is turmoil and a struggle. The hacking compounds things. 

Navigation is always an issue when the conditions are heightened (namely, most of the time). I couldn't cover the 'fin' and very much need the onboard satellite navigation. 

The hackers' remote device has altered settings in the Golf's chassis and engine canbus. 

As mentioned the car shudders over bumps, porpoises over undulations and has poor body control. There's also excessive noise through steering and pedals. 

They've previously switched it on then off again several days later using a remote having device (so it can be deactivated or reversed). It's been activated like this since May 18 2023. 

Please can you help access the canbus so what they've done can be reversed? 

Alternatively please suggest someone else who can please. 

The other option is to provide details of any standalone device that can modify vehicle canbus. 

For me It's not an area of expertise or great understanding. Plus, dexterity issues and constantly heightened disability symptoms make it uncomfortable to experiment without guidance. 

Several resources indicate that various 'can' devices can modify settings.

Please copy and paste the lines below into a browser 

https://ocsaly.com/savvycan-decoding-analyzing-and-modifying-can-data/

Excerpt: 'Savvycan can be used to adjust the timing of the engine and behaviour of the transmission'. 

It stands to reason then that some canbus devices can also adjust chassis and engine canbus.

It would probably be easier to arrange a phone call if you're amenable to it? 

Yours sincerely and many thanks,

Josh Ross

**Comment by simotronic** [comment]
Further possible solutions. The antennae/aerial can be disconnected but
you'd have to determine where the slamm coaxial cable goes.
Or completely disconnecting the car when it's not being supervised.Is it
parked in a private place?
Keeping an accurate log of all the events you experience so perhaps
patterns can be determined.
You say you come from Farringdon? I went to kindergarten the probably can't
call although I suspect that might be easier to discuss any thoughts?
Simon

On Fri, Sep 15, 2023 at 8:53 AM jsr1978 ***@***.***> wrote:

> Thank you for the reply Simon and for recognising gaslighting - it's
> prevalent and disturbing. The hack most definitely is happening and isn't
> imagined.
>
> The car is a 2021 diesel mk8 Golf Gtd. It's a motability car and vital for
> getting around and staying connected with family and friends.
>
> Severe sensory PTSD, agoraphobia and psychogenic hyperventilation means
> much of life is turmoil and a struggle. The hacking compounds things.
>
> Navigation is always an issue when the conditions are heightened (namely,
> most of the time). I couldn't cover the 'fin' and very much need the
> onboard satellite navigation.
>
> The hackers' remote device has altered settings in the Golf's chassis and
> engine canbus.
>
> As mentioned the car shudders over bumps, porpoises over undulations and
> has poor body control. There's also excessive noise through steering and
> pedals.
>
> They've previously switched it on then off again several days later using
> a remote having device (so it can be deactivated or reversed). It's been
> activated like this since May 18 2023.
>
> Please can you help access the canbus so what they've done can be reversed?
>
> Alternatively please suggest someone else who can please.
>
> The other option is to provide details of any standalone device that can
> modify vehicle canbus.
>
> For me It's not an area of expertise or great understanding. Plus,
> dexterity issues and constantly heightened disability symptoms make it
> uncomfortable to experiment without guidance.
>
> Several resources indicate that various 'can' devices can modify settings.
>
> Please copy and paste the lines below into a browser
>
> https://ocsaly.com/savvycan-decoding-analyzing-and-modifying-can-data/
>
> Excerpt: 'Savvycan can be used to adjust the timing of the engine and
> behaviour of the transmission'.
>
> It stands to reason then that some canbus devices can also adjust chassis
> and engine canbus.
>
> It would probably be easier to arrange a phone call if you're amenable to
> it?
>
> Yours sincerely and many thanks,
>
> Josh Ross
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/collin80/SavvyCAN/discussions/668#discussioncomment-7014597>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACYGU7FKZFHSYGKNLM2T3ODX2RMUXANCNFSM6AAAAAA4YLYGIE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
This email, and any attachment to it, may contain information that is
proprietary, privileged or confidential or that may be otherwise legally
exempt from disclosure and is intended only for the individual(s) or entity
to which it is addressed.   If you are not the named recipient, or the
employee or agent responsible for delivering it to the intended recipient,
you are not authorized to read, print, retain, copy, disclose or distribute
this email or any part of it.
If you have received this email in error, please return it immediately to
the sender, delete it and all copies from your system, and destroy any hard
copies of this communication.

**Reply by ghost** [reply]
Thank you Simon. It would be much appreciated if a call can be arranged. Lengthy messages often induce symptoms of the disabilities like tremor and nausea. 


I'd prefer not to disconnect/cover parts of the car when stationary or on the move - the satnav is essential and relied on. I'm relatively au fair with tech. but not an expert and simply desperate for the attacks to end. 


The car is parked next to the house. With certainty the hacker lives in Watford, Herts which is 70 miles away. It's an anti-disability vendetta. 


They are likely using a radio frequency device but I don't know which or where to start with them. 


I remember everything that's happened and have logged much of it. 


My mobile number is +44 7852 239266. Can you provide one so I can call please?


Can a call be arranged or is zoom/Skype preferable? 

I live in Faringdon, Oxfordshire, yes. Attached is a picture of correspondence from the doctor with my address. Where are you located? 

![IMG_20230915_185421_HDR.jpg](https://github.com/collin80/SavvyCAN/assets/145031303/a9e9f109-3b76-46e6-9801-83283f653881)

Help is needed to access the suspension and engine canbus and adjust it back to what it should be. 


If possible I'd also like guidance in how to use the canbus packages. That way if it happens again it can be sorted from home. 


If you can help/guide in how to use the packages and sort the problems I'm happy to agree payment. ![IMG_20230915_185421_HDR.jpg](https://github.com/collin80/SavvyCAN/assets/145031303/3e414fdc-b933-474b-8b07-a5336251c4f1)




Yours sincerely,

Josh Ross

**Reply by ghost** [reply]
Several independent specialists observed the suspension 'bottoming out' on rough sections of road; the wallowing, shuddering and poor body control but couldn't resolve things. It's system not hardware related. Resets haven't helped and main dealers won't accept there is an issue. 

There most definitely is and it's not imagined. It can likely be sorted with canbus access and resending/altering the packets back to what they should be.

All help from the community is much appreciated. 

As the device the hostile hackers with vendettas use isn't known; choosing canbus programmes to access the cars systems is the only other option  ![IMG_20230915_191929_HDR.jpg](https://github.com/collin80/SavvyCAN/assets/145031303/f6d37bce-2342-42fd-b3f6-ac5bed8a76de)

**Comment by afantozzi** [comment]
Hello friend, I follow the savvyCAN project, but I have no knowledge of this tool. However, I would like to help you reset it to factory settings.

I can't help you avoid another attack in the future

You only need to have one J2534 tool And I connect remotely.

If you are interested, get in touch.

**Reply by ghost** [reply]
Hi, thank you for the suggestion and responding. Firstly, can I ask a name please?

I'm a vulnerable disabled 45 year old man with severe sensory PTSD, agoraphobia and psychogenic hyperventilation. 

Before agreeing would a phone call be ok? Also, are there any credentials to share i.e. name, industry?

Help was kindly offered from Simon in Illinois - we spoke on the phone last night. 

As the thread explains this persistent hacking is compounding the very severe disabilities. I'm open to all suggestions that will reverse it's effects/deactivate it. 

A veepack OBD2 scanner will arrive from Amazon on 26 September. Would you need that to access the cars' systems? 

It's very likely that the canbus data for suspension and engine has been altered by the hackers' remote radio frequency device. 

It's a toggle or switch they activate without physical access to the car. The suspension then shudders over bumps, wallows, porpoises over undulations and has poor body control. 

They've adjusted another canbus so that there's excessive noise and vibration through pedals and steering and mpg is 10-20 down. 

Previously they've activated then deactivated several days later so it's not permanent and can be reversed. That said it's now been in this 'mode' since 18 May. 

How deep does the reset suggested go and can you go further by modifying the affected canbus back to what it should be? 

It's had a soft reset at main dealers and fault codes cleared at several independents but still the issues remain. 

Thank you and yours sincerely,

Josh Ross
